x/minfee/module_test.go

```diff
@@ -0,0 +1,41 @@
+package minfee_test
+
+import (
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/v2/x/minfee"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/store"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramkeeper "github.com/cosmos/cosmos-sdk/x/params/keeper"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/stretchr/testify/require"
+	tmdb "github.com/tendermint/tm-db"
+)
+
+func TestNewModuleInitializesKeyTable(t *testing.T) {
+	storeKey := sdk.NewKVStoreKey(paramtypes.StoreKey)
+	tStoreKey := storetypes.NewTransientStoreKey(paramtypes.TStoreKey)
+
+	// Create the state store
+	db := tmdb.NewMemDB()
+	stateStore := store.NewCommitMultiStore(db)
+	stateStore.MountStoreWithDB(storeKey, storetypes.StoreTypeIAVL, db)
+	stateStore.MountStoreWithDB(tStoreKey, storetypes.StoreTypeTransient, nil)
+	require.NoError(t, stateStore.LoadLatestVersion())
+
+	registry := codectypes.NewInterfaceRegistry()
+
+	// Create a params keeper
+	paramsKeeper := paramkeeper.NewKeeper(codec.NewProtoCodec(registry), codec.NewLegacyAmino(), storeKey, tStoreKey)
+	subspace := paramsKeeper.Subspace(minfee.ModuleName)
+
+	// Initialize the minfee module which register set the key table
```

Comment:
```suggestion
	// Initialize the minfee module which registers the key table
```

Reply:
it's very strange to me how sometimes i push a commit fixing/refactoring a piece of code that you still see later on. for example i fixed that comment here https://github.com/celestiaorg/celestia-app/pull/3571/commits/563355e919493e4923b8419b1eae4320f4aad064 and somehow it still showed you an old version 🤔

Reply:
oh hmm I have no idea. I've seen that happen if I start reviewing a PR while the author pushes a commit and I leave a review before refreshing to view what they pushed after I started reviewing.

But in this case it looks like you pushed that commit 3 hours ago and I reviewed ~24 mins ago so I should've seen the latest changes 🤷